Source/WebKit/DerivedSources.make

```diff
@@ -400,6 +400,9 @@ all : $(SANDBOX_PROFILES_WITHOUT_WEBPUSHD) $(WEBPUSHD_SANDBOX_PROFILE) $(SANDBOX
 %.sb : %.sb.in
 	@echo Pre-processing $* sandbox profile...
 	grep -o '^[^;]*' $< | $(CC) $(SDK_FLAGS) $(TARGET_TRIPLE_FLAGS) $(SANDBOX_DEFINES) $(TEXT_PREPROCESSOR_FLAGS) $(FRAMEWORK_FLAGS) $(HEADER_FLAGS) $(EXTERNAL_FLAGS) -include "wtf/Platform.h" - > $@
+	if [ -e "/usr/local/bin/sbutil" ]; then \
+		sbutil compile -D HOMEDIR=homedir -D IMPORT_DIR=$(SDKROOT)/usr/local/share/sandbox/profiles/embedded/imports $@ \
```

Comment:
Same as before, this shouldn't spew binary output into the build log.

Also, I'm assuming `HOMEDIR=homedir` is a fake directory, just needed to fill in a sandbox parameter?

Reply:
Oh, and the build error is because Make is escaping the newline on this line. So you need a `;` before the `fi`.

Reply:
> Same as before, this shouldn't spew binary output into the build log.
> 
> Also, I'm assuming `HOMEDIR=homedir` is a fake directory, just needed to fill in a sandbox parameter?

Yes.

Reply:
> Oh, and the build error is because Make is escaping the newline on this line. So you need a `;` before the `fi`.

Ah, thanks, will fix!